JetUtilities/src/L2Creator.cc

```diff
@@ -43,6 +45,7 @@ L2Creator::L2Creator(CommandLine& cl) {
   mpv        = cl.getValue<bool>    ("mpv",            false);
   delphes    = cl.getValue<bool>    ("delphes",        false);
   maxFitIter = cl.getValue<int>     ("maxFitIter",        30);
+  modified_fdscb= cl.getValue<bool> ("modified_fdscb", false);
```

Comment:
Yes, but it's not working (I spend time to figure this out). Please keep it this way (at last for now).